CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+ 
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+ 
+## [Unreleased] - yyyy-mm-dd
```

Comment:
@karenkrieger a ideia é manter o changelog sempre manual ou ter algo mais automatizado no futuro tipo [isso](https://www.freecodecamp.org/news/a-beginners-guide-to-git-what-is-a-changelog-and-how-to-generate-it/)?